.github/workflows/version-update.yml

```
@@ -0,0 +1,77 @@
+#
+# Copyright 2025 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: update-dapr-version
+
+on:
+  workflow_dispatch:
+    inputs:
+      rel_version:
+        description: 'Dapr release version (examples: 1.9.0-rc.1, 1.9.1)'
+        required: true 
+        type: string
+
+jobs:
+  update-longhauls:
+    name: Update the dapr version in longhaul tests
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    env:
+      LONG_HAUL_REPO: test-infra
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+      - name: Checkout the dapr repo
+        uses: actions/checkout@v4
+        with:
+          repository: dapr/dapr
+          path: .dapr_repo
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install pcre2-utils
+          pip install packaging
```

```suggestion
          pip install semver
```

I suggest we move the `compare_versions.py` file from the dapr repo to this repo and update it so it uses the semver package instead of the `packaging` one that uses the PEP440 standard instead of semver.